rapid7_insightvm/Dockerfile

```diff
@@ -7,6 +7,23 @@ ENV SSL_CERT_FILE /etc/ssl/certs/ca-certificates.crt
 ENV SSL_CERT_DIR /etc/ssl/certs
 ENV REQUESTS_CA_BUNDLE  /etc/ssl/certs/ca-certificates.crt
 
+# Needed for maya
+RUN apk add --no-cache --virtual .build-deps \
+                                        make \
+                                        gcc \
+                                        libc-dev \
+                                        libffi-dev \
+                                        openssl-dev \
+                                        libxml2-dev \
+                                        libxslt-dev
+# This is not the proper way to do dependencies usually, however, the latest version of pendulum
+# (which is a maya dependency) has a bug...so to ensure requirements are installed in order
+# I had to do the following
+RUN pip install --upgrade pip
+RUN pip install -U setuptools
+RUN pip install pendulum==2.0.5
+RUN pip install -r requirements.txt
```

Comment:
You can go the `wheels` route with this which will solve a lot of the headaches, if you'd like. The Exchange plugin does this if you want an example.

Reply:
I'll keep that in mind next time I use Maya.